magento/magento2-phpstorm-plugin
# Line markers support for target plugin class and methods

### Description (*)
This PR introduces line markers support for target plugin class and methods. So you are able to navigate to a target class of the plugin or a target method of the plugin method directly from the plugin 

![image](https://user-images.githubusercontent.com/2148959/78449964-7093a280-767b-11ea-9140-e410699ee470.png)


![image](https://user-images.githubusercontent.com/2148959/78449966-7b4e3780-767b-11ea-8fd6-e1f3378efce6.png)


### Fixed Issues (if relevant)
1. #61 : Line Marker. Add Plugin Method -> Target Method line marker

### Manual Testing Scenarios
1. Install the plugin 
2. Open some existing plugin
3. Make sure that you are able to navigate to plugin target class/method
